code/modules/atmospherics/machinery/components/binary_devices/pump.dm

```diff
@@ -27,6 +27,13 @@
 	construction_type = /obj/item/pipe/directional
 	pipe_state = "pump"
 
+/obj/machinery/atmospherics/components/binary/pump/CtrlClick(mob/user)
+	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+		on = !on
+		target_pressure = MAX_OUTPUT_PRESSURE
```

Comment:
you don't always want a pump to have full pressure

Reply:
99% of time you want to and if you want to set it to a specific pressure you can do it manually

Reply:
> if you want to set it to a specific pressure you can do it manually

I 100% agree so make it so.

Reply:
i havent removed the old way of setting the pump (click and then set it) so its already in ?

Reply:
but then you can't ever use the quick function to turn it off or on because your block forces it at a set pressure when people should manually set it.

please take it out.

Reply:
dude no one sets it to "a certain pressure" and having it turn on and then you have to click max pressure invalidates the whole shortcut useless as you have to open the dumb window anyway
if you want to set a pump to 222 for some reason you dont use the shortcut 
if you want to set the supermatter you ctrl click instead of opening 5 laggy  pop up windows every fucking round

Reply:
"dude" if you dislike the default pressures.... crazy idea.... change the default pressures.

don't put it in here lol.